### PR TITLE
Wizard recipe: Fastjet-v3.3.3

### PR DESCRIPTION
**Pull request by jstrube**
This pull request contains a new build recipe I built using the BinaryBuilder.jl wizard:

* Package name: Fastjet
* Version: v3.3.3

@staticfloat please review and merge.

### CODE REVIEW

File: F/Fastjet/build_tarballs.jl [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+cd fastjet-3.3.3/
+./configure --prefix=${prefix} --build=${MACHTYPE} --host=${target}
+make -j ${nprocs}
+make install
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = [
+    Linux(:x86_64, libc=:musl),
```
[the remark this answers is not in the document]
Comment (jstrube) [reply F/Fastjet/build_tarballs.jl]: I'd be happy to build this on as many platforms as possible. Thank you for testing.

[the remark this answers is not in the document]
Comment (giordano) [reply F/Fastjet/build_tarballs.jl]: Can I modify this script?

[the remark this answers is not in the document]
Comment (jstrube) [reply F/Fastjet/build_tarballs.jl]: Please do! Thank you!

[the remark this answers is not in the document]
Comment (jstrube) [reply F/Fastjet/build_tarballs.jl]: That libtool issue on Windows is beyond my ability to fix...

**Comment by jstrube** [comment]
That libtool issue on Windows in beyond my ability to fix...

**Comment by giordano** [comment]
I'll have a look later

**Comment by giordano** [comment]
If you're happy with the changes, we should be good to go

**Comment by jstrube** [comment]
You're a wizard

**Comment by giordano** [comment]
I assume that's a "looks good" :stuck_out_tongue:

**Comment by jstrube** [comment]
lgtm

**Pull request merged**